Dividing contributors guide into expert and beginner parts (#13742)
mistercrunch/Airflow: COMMITTERS.rst

.. Licensed to the Apache Software Foundation (ASF) under one
   or more contributor license agreements.  See the NOTICE file
   distributed with this work for additional information
   regarding copyright ownership.  The ASF licenses this file
   to you under the Apache License, Version 2.0 (the
   "License"); you may not use this file except in compliance
   with the License.  You may obtain a copy of the License at

 ..   http://www.apache.org/licenses/LICENSE-2.0

 .. Unless required by applicable law or agreed to in writing,
    software distributed under the License is distributed on an
    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
    KIND, either express or implied.  See the License for the
    specific language governing permissions and limitations
    under the License.

.. contents:: :local:

> Committers and PMS's

Reviewer: ```suggestion
Committers and PMC's
```

====================

> This document assumes that you know how our community works in our community, but you would like to learn more about the rules by which we add new members.

Reviewer: ```suggestion
This document assumes that you know how Airflow's community work, but you would like to learn more about the rules by which we add new members.
```

> Before reading this document, you should be familiar with `Contributor guides <https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst>`__.

Reviewer: ```suggestion
Before reading this document, you should be familiar with `Contributor's guide <https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst>`__.
```
